### PR TITLE
Update the ping test so it will grab the url from the ping server

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by nojunpark** [comment]
@JohanLorenzo r?  this now fetches the remote url from the ping server, instead of verifying whether it matches the parameter initially provided.

File: test/integration/test/no_travis/pagekite_ping_test.js [hunk 1]
```diff
@@ -7,59 +7,69 @@ var config = new Config('./test/integration/lib/config/foxbox.js');
...
-    }
-    local_ip = pingResp.body[pick].local_ip;
-    tunnel_url = pingResp.body[pick].tunnel_url;
-
-    expect(pingResp).to.have.status(200);
-    expect(local_ip).to.match(/\d+.\d+.\d+.\d+/);
-    expect(tunnel_url).equals(config.get('pagekite.externalURL'));
-    expect(pingResp.body[pick].timestamp).to.match(/\d+/);
-    expect(pingResp.body[pick].public_ip).to.match(/\d+.\d+.\d+.\d+/);
+      var originList = JSON.parse(pingResp.body[pick].message);
```
Comment (JohanLorenzo) [inline test/integration/test/no_travis/pagekite_ping_test.js]: Nit: Indentation

File: test/integration/test/no_travis/pagekite_ping_test.js [hunk 2]
```diff
@@ -7,59 +7,69 @@ var config = new Config('./test/integration/lib/config/foxbox.js');
...
+      var pick;
+      var timestamp = 0;
+
+      return  chakram.get(pingUrl,{'json': true})
+      .then(function(pingResp){
+        for (var match in pingResp.body) {
           // may be multiple entries.  in that case, pick latest
           if (parseInt(pingResp.body[match].timestamp) > 
             parseInt(timestamp)) {
             timestamp = pingResp.body[match].timestamp;
```
Comment (JohanLorenzo) [inline test/integration/test/no_travis/pagekite_ping_test.js]: Nit: Not used anymore

File: test/integration/test/no_travis/pagekite_ping_test.js [hunk 3]
```diff
@@ -7,59 +7,69 @@ var config = new Config('./test/integration/lib/config/foxbox.js');
...
+  });
+  
+  describe('Initiate the remote connection',function(){
+    before('fetch the url', function() {
+      var pick;
+      var timestamp = 0;
+
+      return  chakram.get(pingUrl,{'json': true})
+      .then(function(pingResp){
+        for (var match in pingResp.body) {
```
Comment (JohanLorenzo) [inline test/integration/test/no_travis/pagekite_ping_test.js]: Nit: this could be extracted in a function called like `getLatestIp(body)`.

**Comment by JohanLorenzo** [comment]
Looks good to me, modulo a couple of nits

File: test/integration/test/no_travis/pagekite_ping_test.js (hunk 2 above)
Comment (nojunpark) [inline test/integration/test/no_travis/pagekite_ping_test.js]: actually, timestamp value is used to compare the time value of the previously most recent entry to the current entry.

Comment (JohanLorenzo) [inline test/integration/test/no_travis/pagekite_ping_test.js]: Oopsie, you're totally right. :sweat_smile:

**Pull request merged**